### PR TITLE
📝 Adds Metabase introduction

### PR DESCRIPTION
**Pull request by cynektix**

### CODE REVIEW

File: _posts/2017-04-06-introduction-to-metabase.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+When you start a company, setting up a business intelligence (BI) solution is not the first thing most founding teams
+implement, I guess. Now that I'm passed the step of implementing such a solution, I think it is something that should
+be implemented in the very early days to build a culture of measuring everything.
+
+The main reason why it wasn't implemented directly from the beginning at my company was that no one had the
+experience with a good solution yet. Therefore, reports were conducted manually against the database in raw SQL
+or implemented into the internal admin area to make them a little bit more accessible (but without graphs of course
+because it would have cost to much time). As you might guess, this was some work and couldn't be changed or extended fast.
```
Comment (tomhoule) [inline _posts/2017-04-06-introduction-to-metabase.md]: typo ("too much")

File: _posts/2017-04-06-introduction-to-metabase.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Using the production PostgreSQL user creates a security risk because this user can change data in the database. As
+Metabase is only for reporting, and therefore reading data, we created a read-only PostgreSQL user for our
+production database.
+
+It is not a good idea to give Metabase direct access to your production database. When you create huge and slow queries,
+it will directly affect your production servers. Fortunately, [AWS RDS makes it super easy to create a read replica
+of your production database.](https://aws.amazon.com/rds/details/read-replicas/)
+
+Metabase needs a database it can read and write from for itself in order to manage data about the users, reports and so
+on. To make the database of Metabase persistent accress Kubernetes deployments, we created another database on our
```
Comment (tomhoule) [inline _posts/2017-04-06-introduction-to-metabase.md]: typo

File: _posts/2017-04-06-introduction-to-metabase.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  - Metabase
+  - BI
+  - PostgreSQL
+author_staff_member: 01_peter
+medium_link:
+practical_dev_link:
+---
+
+When you start a company, setting up a business intelligence (BI) solution is not the first thing most founding teams
+implement, I guess. Now that I'm passed the step of implementing such a solution, I think it is something that should
```
Comment (brainlessdeveloper) [inline _posts/2017-04-06-introduction-to-metabase.md]: `Now that I'm past*`

File: _posts/2017-04-06-introduction-to-metabase.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+or implemented into the internal admin area to make them a little bit more accessible (but without graphs of course
+because it would have cost too much time). As you might guess, this was some work and couldn't be changed or extended fast.
+
+In the fall of 2015, [AWS QuickSight was announced](https://aws.amazon.com/blogs/aws/amazon-quicksight-fast-easy-to-use-business-intelligence-for-big-data-at-110th-the-cost-of-traditional-solutions/).
+It seemed to be a very good solution for us as our databases are hosted on AWS. In August 2016 we got access to
+the preview version. It wasn't really easy to use and the user management was not really good. Reports and datasources
+weren't shared accross users or I didn't find the right way to do it. In most cases we use UUIDs as our primary keys and
+AWS QuickSight had no support for UUIDs back then. This lead to the situation that I didn't put more time and effort in
+using AWS QuickSight. When [AWS QuickSight was finally released](https://aws.amazon.com/blogs/aws/amazon-quicksight-now-generally-available-fast-easy-to-use-business-analytics-for-big-data/),
+I took another look at it but I still wasn't satified with the look and feel.
```
Comment (brainlessdeveloper) [inline _posts/2017-04-06-introduction-to-metabase.md]: `satisfied`

File: _posts/2017-04-06-introduction-to-metabase.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+## First steps
+
+I really recommend [reading the docs of Metabase](http://www.metabase.com/docs/v0.23.1/). They are really good,
+especially for the first steps.
+
+In Metabase reports are called questions. So in order to create the first report you click on `New Question`, select
+the database, select the table, add filters, specify what you want to see (eg. count of users) and add a grouping
+(eg. created_at by month). And all of that in a simple click interface that can be used by a lot of users and not only the
+SQL pros in the company. (It still gives you the possibility to create custom SQL queries.)
```
Comment (brainlessdeveloper) [inline _posts/2017-04-06-introduction-to-metabase.md]: `...to create custom SQL queries).`

**Comment by brainlessdeveloper** [comment]
Very nice, I did some nitpicking and found some typos

**Pull request merged**